Shuttlerock/actions
# [STUDIO-1150] Sum story points of child issues in the parent issue

## Sum story points of child issues in the parent issue

[Jira Story](https://shuttlerock.atlassian.net/browse/STUDIO-1150)
[Jira Epic](https://shuttlerock.atlassian.net/browse/STUDIO-231)



## How to test the PR

Add an estimate to a issue story, wait 30 seconds for the action to run, and refresh the parent issue - check that the `Story points estimate` field has been updated.

```
$ act --job trigger_action --eventpath src/actions/trigger-action/__tests__/fixtures/jiraStoryPointsUpdated.json
```

## Deployment Notes

None
